package.json

```diff
@@ -11,9 +11,9 @@
     "build:holochain": "VUE_APP_UI_VERSION=$npm_package_version VUE_APP_CONTEXT='holochain' vue-cli-service build",
     "test:unit-hc": "VUE_APP_CONTEXT='holochain' vue-cli-service test:unit --watch=true --config=./jest.hcunit.config.js",
     "test:unit-holo": "VUE_APP_CONTEXT='holo-host' vue-cli-service test:unit --watch=true --config=./jest.holounit.config.js",
-    "test-setup:happ": "nix-shell --command 'yarn fetch:dna && yarn build:happ' && exit",
-    "test-setup:holo": "yarn run clear:hc-holo && nix-shell --command 'yarn hc-holo:gen-agent && yarn hc-holo:run && yarn run:test-chaperone-hcc' && exit",
-    "test:integration-hc": "yarn run clear:hc && yarn run clear:lair && yarn build:holochain && nix-shell --command 'VUE_APP_CONTEXT='holochain' VUE_APP_WEB_LOGS='false' VUE_APP_TEST_PORT='8888' node scripts/test.js --config=./jest.hcintegration.config.js --watch --runInBand --detectOpenHandles --testTimeout=95000000' && exit",
+    "test-setup:happ": "yarn fetch:dna && yarn build:happ",
+    "test-setup:holo": "yarn run clear:hc-holo && yarn hc-holo:gen-agent && yarn hc-holo:run && yarn run:test-chaperone-hcc",
+    "test:integration-hc": "yarn run clear:hc && yarn run clear:lair && yarn build:holochain && VUE_APP_CONTEXT='holochain' VUE_APP_WEB_LOGS='false' VUE_APP_TEST_PORT='8888' node scripts/test.js --config=./jest.hcintegration.config.js --watch --runInBand --detectOpenHandles --testTimeout=95000000",
```

Comment:
I don't use nix-shell for EC UI development so I had to remove the forced usages of it. Let me know if you need me to add them back

Reply:
Actually, yes, please do restore them - we need nix-shell for the ci to build and test with the correct version of holochain.